mpa/modules/models/heads/cross_dataset_detector_head.py

```diff
@@ -236,4 +217,15 @@ def vfnet_to_atss_targets(self,
         ]
         label_weights = torch.cat(label_weights_list)
         bbox_weights = torch.cat(bbox_weights_list)
-        return labels_list, label_weights, bbox_targets_list, bbox_weights
+        return labels_list, label_weights, bbox_targets_list, bbox_weights, ignored_masks
+
+    def get_ignored_masks(self, img_metas, all_labels):
+        ignored_masks = []
+        for i, meta in enumerate(img_metas):
+            mask = torch.Tensor([1 for _ in range(self.num_classes)])
+            if 'ignored_labels' in meta and meta['ignored_labels']:
+                mask[meta['ignored_labels']] = 0
+            mask = mask.repeat(len(all_labels[i]), 1)
```

Comment:
Do you need this code? What is the purpose?

Reply:
@chuneuny-emily Yes, this is a function that takes ignored_labels and makes them a mask that will be used for loss. Because information about which images each label belongs to is not passed to loss, the ignored_labels are replaced with masks in the head.

Reply:
> @chuneuny-emily Yes, this is a function that takes ignored_labels and makes them a mask that will be used for loss. Because information about which images each label belongs to is not passed to loss, the ignored_labels are replaced with masks in the head.

I understood.